README.md

```diff
@@ -102,6 +130,10 @@
 
 > - GitHub: [hydaline](https://github.com/hydaline-code)
 
+👤 **Omet Omeni**
+
+> - GitHub: [hydaline](https://github.com/ometman)
+
```

Comment:
[REQUIRED] There are no acknowledgment fields and MIT section on your `README.md`, kindly add them and ensure you link the `MIT` properly to the LICENSE file you already have.

Reply:
@VICTOR-CHUKWU please which styling is it css please  how to make the signup form looks like login form?
r